docs/docs/working-with-images-in-markdown.md

```diff
@@ -167,7 +167,36 @@ module.exports = {
 }
 ```
 
-With this configuration, you can use the default Markdown syntax for images. They will be processed by Sharp and appear as if you placed them in a `gatsby-image` component.
+Here is a similar example using the `gatsby-transformer-remark` plugin instead of `gatsby-plugin-mdx`.
```

Comment:
I can definitely see this being useful; it would help to break the sections into two subheadings. Can you add an h3 (`###`) for each section for the different inline image configurations?

Reply:
No problem! The headings will definitely help to break up the sections better.